spec/features/adventures/user_searches_adventures_spec.rb

```diff
@@ -0,0 +1,367 @@
+  #   )
+  #   bucket_list_adventure_1 = FactoryGirl.create(
+  #     :bucket_list_adventure,
+  #     adventure_id: adventure_1.id,
+  #     bucket_list_id: bucket_list_1.id
+  #   )
+  #
+  #   bucket_list_2 = FactoryGirl.create(
+  #     :bucket_list,
+  #     user_id: user.id,
+  #     title: "Mexico City"
+  #   )
+  #   adventure_2 = FactoryGirl.create(:adventure,
+  #     user_id: user.id,
+  #     name: "Num. 2",
+  #     is_shared: false
+  #   )
+  #   bucket_list_adventure_2 = FactoryGirl.create(
+  #     :bucket_list_adventure,
+  #     adventure_id: adventure_2.id,
+  #     bucket_list_id: bucket_list_2.id
+  #   )
+  #
+  #   bucket_list_3 = FactoryGirl.create(
+  #     :bucket_list,
+  #     user_id: user.id,
+  #     title: "Guatamala"
+  #   )
+  #   adventure_3 = FactoryGirl.create(:adventure,
+  #     user_id: user.id,
+  #     name: "Num. 3"
+  #   )
+  #   bucket_list_adventure_3 = FactoryGirl.create(
+  #     :bucket_list_adventure,
+  #     adventure_id: adventure_3.id,
+  #     bucket_list_id: bucket_list_3.id
+  #   )
+  #
+  #   visit bucket_lists_path
+  #   find('.adventures-search-bar').set("mexico")
+  #   click_button "Find Adventure!"
+  #
+  #   expect(page).to have_content(adventure_1.name)
+  #   expect(page).to have_content(adventure_2.name)
+  #   expect(page).not_to have_content(adventure_3.name)
+  # end
+
+  scenario "authenticated user sucessfully searches adventures and discovers " \
+  "other user's shared adventure" do
+    current_user = FactoryGirl.create(:user)
+    other_user = FactoryGirl.create(:user, username: "OtherUser111")
+    sign_in(current_user)
+
+    bucket_list_1 = FactoryGirl.create(
+      :bucket_list,
+      user_id: current_user.id,
+    )
+    adventure_1 = FactoryGirl.create(:adventure,
+      user_id: current_user.id,
+      is_shared: true,
+      address: "Temple Bar, Beijing, China"
+    )
+    bucket_list_adventure_1 = FactoryGirl.create(
+      :bucket_list_adventure,
+      adventure_id: adventure_1.id,
+      bucket_list_id: bucket_list_1.id
+    )
+
+    bucket_list_2 = FactoryGirl.create(
+      :bucket_list,
+      user_id: other_user.id,
+    )
+    adventure_2 = FactoryGirl.create(:adventure,
+      user_id: other_user.id,
+      is_shared: true,
+      name: "Beijing Undergound City"
+    )
+    bucket_list_adventure_2 = FactoryGirl.create(
+      :bucket_list_adventure,
+      adventure_id: adventure_2.id,
+      bucket_list_id: bucket_list_2.id
+    )
+
+    bucket_list_3 = FactoryGirl.create(
+      :bucket_list,
+      user_id: other_user.id
+    )
+    adventure_3 = FactoryGirl.create(:adventure,
+      user_id: other_user.id,
+      is_shared: false,
+      address: "D-22, Beijing China"
+    )
+    bucket_list_adventure_3 = FactoryGirl.create(
+      :bucket_list_adventure,
+      adventure_id: adventure_3.id,
+      bucket_list_id: bucket_list_3.id
+    )
+
+    visit bucket_lists_path
+    find('.adventures-search-bar').set("beijing")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.